### PR TITLE
Jack test

### LINKED ISSUE #12: Last Queries For Application Description
- [x] Modify Query for Survey Site to Sort by Pollutant Type first
- [x] Add Query for Querying a State in which we sort by Survey Site and then a particular pollutant

### PR DESCRIPTION
**Pull request by jsummers1221**
BySite now orders by pollutant name and date. ByState now joins with the pollutant info and selects the pollutant name.

### CODE REVIEW

File: sqlQueries/queryPollutantByState.sql [hunk 1]
```diff
@@ -15,11 +19,15 @@ ORDER BY date_local
 --Returns all samples for a state
```
Comment (cai-michael) [inline sqlQueries/queryPollutantByState.sql:19]: SELECT psample.date_local, psample.id, ptype.name, AVG(psample.mean)
FROM pollutant_sample AS psample
INNER JOIN taken_at AS ta
	ON psample.sample_id = ta.sample_id
INNER JOIN in_state
	ON ta.site_num = in_state.site_num
INNER JOIN is_type AS it
    ON psample.id = it.sample_id
INNER JOIN pollutant_type AS ptype
    ON ptype.id = it.type_id
WHERE in_state.state_code = user_provided_state_code
GROUP BY date_local
ORDER BY ptype.name, date_local`

**Review by cai-michael — CHANGES_REQUESTED** [review]

File: sqlQueries/queryPollutantByState.sql (hunk 1 above)
Comment (jsummers1221) [reply sqlQueries/queryPollutantByState.sql:19]: Added this to queryPollutantByState.sql (at the end of the file):
--Returns the average of the means of the pollutants across all sites in a state, for each date.
--The data is ordered by date_local and pollutant name which is useful for creating a linegraph
--Inputs (user_provided_state_code)
SELECT psample.date_local, psample.id, ptype.name, AVG(psample.mean)
FROM pollutant_sample AS psample
	INNER JOIN taken_at AS ta
		ON psample.sample_id = ta.sample_id
	INNER JOIN in_state
		ON ta.site_num = in_state.site_num
	INNER JOIN is_type AS it
		ON psample.id = it.sample_id
	INNER JOIN pollutant_type AS ptype
		ON ptype.id = it.type_id
WHERE in_state.state_code = user_provided_state_code
GROUP BY date_local, ptype.name
ORDER BY ptype.name, date_local;

Comment (jsummers1221) [reply sqlQueries/queryPollutantByState.sql:19]: I restored queryPollutantByState to what it was before the averages were used, and made averaged versions of those queries in queryPollutantByState_Average.

**Review by cai-michael — APPROVED** [review]
Looks Good!

**Pull request merged**